PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -351,6 +366,56 @@ To add more Modules, with the understanding that this should be done with cautio
 
 To edit an existing Program, either find the name and click _Edit_ in the options in the right column from the list view or from the _Show_ page, click the EDIT button at the top of the screen. Click on the SAVE button once all the desired information has been changed or entered.
 
+# System Settings Configuration
+
+System settings (```SystemSettings```) is part of the configuration bundle and is a collection of settings that affect the whole build.
+
+## Age Range
+
+This is for developers working on configuration files. Age range is set in the ```SystemSettings``` part of the configuration bundle by the ```primary_age_range``` property.
+
+* There are two arrays of age ranges primero and unhcr with unhcr being the default set by the ```age_ranges``` property
+
+* The default age ranges are ```["0..4", "5..11", "12..17", "18..59", "60..#{AgeRange::MAX}"]```
+
+## Reporting Location
+
+In ```SystemSettings``` under ```reporting_location_config``` the ```hierarchy_filter``` replaces ```reg_ex_filter``` to filter locations. This should match the hierarchy array in ```Locations```
+
+## Agency Code
+
+This is for developers working on configuration files. The **Agency_code** is added to the "Case" on the top left of the case view (for example "FPD Case"). This is defined in the Agency lookup as the ```agency_code``` property and the display of the agency code is toggled by the module property ```agency_code_indicator``` (set to true by defalt).
+
+## This is for developers working on configuration files. Search for an existing ID before creating a Case record
```

Comment:
Once again, don't have this in the header. Instead, keep the header as it was, and put the following as a note at the beginning of the paragraph: "(This section is for developers editing configuration files or working on the configuration repository.)"